### DISCUSSION TITLE
node-red-contrib-watt2kwh

### DISCUSSION THREAD

**Discussion by erottensteiner** [rdmtc/RedMatic#583] [Q&A]
Hallo,
ich habe heute diese neue Node installiert, und zwar direkt über putty mit "npm install node-red-contrib-watt2kwh".
Auch nach einem Reboot von RedMatic scheint diese neue Node nicht in der Übersicht auf - ich habe als Gegenprobe auch "node-red-contrib-shelly" installiert - diese Nodes sind nach der Installation auch sichtbar und verfügbar.

Wenn ich "node-red-contrib-watt2kwh" versuche, direkt in Redmatic/Node-Red unter "Palette verwalten" zu installieren, gibt's allerdings einen Fehler "Installation fehlgeschlagen: node-red-contrib-watt2kwh - Cannot read property 'nodes' of null - Überprüfen Sie das Protokoll auf weitere Informationen." - und auch das Log gibt nicht viel mehr Infos her:
2023-10-29T20:18:59.200Z installieren : node-red-contrib-watt2kwh 0.1.2
2023-10-29T20:18:59.315Z npm install --no-package-lock --global-style --no-audit --no-update-notifier --no-fund --save --save-prefix=~ --production node-red-contrib-watt2kwh@0.1.2
2023-10-29T20:19:06.269Z [out] 
2023-10-29T20:19:06.269Z [out] up to date in 6s
2023-10-29T20:19:06.318Z rc=0

Bräuchte diese node um aus aktuellen Watt-Werten kWh-Tagesmengen berechnen zu lassen...
Hat hier jemand vielleicht einen Tipp oder schon Erfahrung mit dieser Node gesammelt bzw. auch eine ev. Alternative?

Viele Grüße

**Comment by Tscherno** [comment]
Da RedMatic leider tot ist: HomeAssistant und dort NodeRed oder direkt in HA umsetzen.

**Comment by hobbyquaker** [comment]
Nodes werden über den Paletten-Manager von Node-RED installiert bzw. auf der Kommandozeile im Verzeichnis var nach source /usr/local/addons/redmatic/home/.profile - siehe https://github.com/rdmtc/RedMatic/wiki/Node-Installation. RedMatic 9 ist verfügbar: https://github.com/rdmtc/RedMatic/releases. Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
